govcd/media.go

```diff
@@ -337,3 +337,20 @@ func FindMediaAsCatalogItem(org *Org, catalogName, mediaName string) (CatalogIte
 	}
 	return media, nil
 }
+
+// Refresh refreshes the media item information by href
+func (mediaItem *MediaItem) Refresh() error {
+
+	if mediaItem.MediaItem == nil {
+		return fmt.Errorf("cannot refresh, Object is empty")
+	}
+
+	url := mediaItem.MediaItem.HREF
```

Comment:
Shouldn't we error if the HREF is not set?

Reply:
fixed